### PR TITLE
Add travis file

### PR DESCRIPTION
**Pull request by matevzmihalic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: go
+go:
+  - "1.10.x"
+sudo: true
+
+before_install:
+  - sudo add-apt-repository -y ppa:maarten-fonville/protobuf
```
Comment (dominikznidar) [inline .travis.yml]: protobuf can probably removed from the project

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - sudo apt-get -qq update
+  - sudo apt-get install -qq protobuf-compiler
+  - go get github.com/golang/mock/mockgen
+           github.com/golang/protobuf/protoc-gen-go
+           github.com/dominikznidar/govendor
+           github.com/go-swagger/go-swagger/cmd/swagger
+  - make
+
+script:
+  - go test -short ./...
```
Comment (dominikznidar) [inline .travis.yml]: Why not `make test`?

Comment (matevzmihalic) [reply .travis.yml]: To avoid 6 minutes of `storage/encrypted_bolt` tests.

Should I change `make test` to run short test?

Comment (dominikznidar) [reply .travis.yml]: yes

**Comment by dominikznidar** [comment]
👍

**Pull request merged**